boruta/boruta/script.whizzml

```diff
@@ -0,0 +1,362 @@
+;; This will eventually be packed into a library
```

Comment:
i guess all the comments above apply to this file too.  do we really need all this duplication? why not include only the 1-click, and we'll refactor to library when we can?